specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/virtualNetwork.json

```diff
@@ -1081,13 +1081,17 @@
           "type": "string",
           "description": "The provisioning state of the PublicIP resource. Possible values are: 'Updating', 'Deleting', and 'Failed'."
         },
-        "enableDdosProtection": { 
-          "type": "boolean", 
-          "description": "Indicates if DDoS protection is enabled for all the protected resources in a Virtual Network." 
-        }, 
-        "enableVmProtection": { 
-          "type": "boolean", 
-          "description": "Indicates if Vm protection is enabled for all the subnets in a Virtual Network." 
+        "enableDdosProtection": {
+          "type": "boolean",
+          "description": "Indicates if DDoS protection is enabled for all the protected resources in the virtual network. It requires a DDoS protection plan associated with the resource."
+        },
+        "enableVmProtection": {
+          "type": "boolean",
```

Comment:
Is there a default value?

Reply:
Also false by default

Reply:
...it would be a good idea to add the default value to the swagger. This way you will get correct documentation.